src/lib/snack-bar/snack-bar-animations.ts

```diff
@@ -13,27 +13,19 @@ import {
   trigger,
   AnimationTriggerMetadata,
 } from '@angular/animations';
-import {AnimationCurves, AnimationDurations} from '@angular/material/core';
 
 /** Animations used by the Material snack bar. */
 export const matSnackBarAnimations: {
-  readonly contentFade: AnimationTriggerMetadata;
   readonly snackBarState: AnimationTriggerMetadata;
 } = {
-  /** Animation that slides the dialog in and out of view and fades the opacity. */
-  contentFade: trigger('contentFade', [
-    transition(':enter', [
-      style({opacity: '0'}),
-      animate(`${AnimationDurations.COMPLEX} ${AnimationCurves.STANDARD_CURVE}`)
-    ])
-  ]),
-
   /** Animation that shows and hides a snack bar. */
   snackBarState: trigger('state', [
-    state('visible-top, visible-bottom', style({transform: 'translateY(0%)'})),
-    transition('visible-top => hidden-top, visible-bottom => hidden-bottom',
-      animate(`${AnimationDurations.EXITING} ${AnimationCurves.ACCELERATION_CURVE}`)),
-    transition('void => visible-top, void => visible-bottom',
-      animate(`${AnimationDurations.ENTERING} ${AnimationCurves.DECELERATION_CURVE}`)),
+    state('void', style({opacity: 0, transform: 'scale(0.8)'})),
```

Comment:
Does it make sense to have styles in the `transition()` calls?

Could this just be:
```ts
 snackBarState: trigger('state', [
    state('void', style({
      opacity: 0,
      transform: 'scale(0.8)'
    })),
    state('visible', style({
      transform: 'scale(1)',
      opacity: 1,
    })),
    transition('* => visible', animate('150ms cubic-bezier(0, 0, 0.2, 1)')),
    transition('* => void', animate('75ms cubic-bezier(0.4, 0.0, 1, 1)')),
  ])
```

Reply:
Not exactly. The code above will cause it to scale away for the exit animation, but we only want it to fade out. This means that we need a `style({opacity: 0})` on the `* => void`. I've changed it with some tweaks.